conf/ocsci/rest_encryption.yaml

```diff
@@ -0,0 +1,6 @@
+---
+# This configuration file is used for enable REST encryption in the deployment
+# Valid only for OCS >= 4.6
+# https://issues.redhat.com/browse/KNIP-1264
+ENV_DATA:
+  rest_encryption: true
```

Comment:
Don't call it **rest encryption**, which could be little puzzling, but use `encryption_at_rest`.